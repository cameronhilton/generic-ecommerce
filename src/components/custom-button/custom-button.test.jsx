import { shallow } from "enzyme";
import CustomButton from "./custom-button.component";

it("renders CustomButton component", () => {
  expect(shallow(<CustomButton />)).toMatchSnapshot();
});

it("renders children when passed in", () => {
  const wrapper = shallow(
    <CustomButton>
      <div className="unique" />
    </CustomButton>
  );
  expect(wrapper.contains(<div className="unique" />)).toBe(true);
});
