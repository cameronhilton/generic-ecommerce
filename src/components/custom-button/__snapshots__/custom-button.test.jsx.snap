// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CustomButton component 1`] = `ShallowWrapper {}`;
